models/JobModel.js

```diff
@@ -1,17 +1,20 @@
-var mongoose = require("mongoose");
+const mongoose = require("mongoose");
 
-var Schema = mongoose.Schema;
+const Schema = mongoose.Schema;
+const DataTypes = Schema.Types;
+const SkillSchema = require("./SkillModel");
 
-var JobSchema = new Schema({
-	title: { type: String, required: true },
-	description: { type: String, required: true },
-	date: { type: Date, required: true },
-	cost: { type: Number, required: true },
-	status: { type: String, required: true },
-	owner: { type: String, required: true },
-	email: { type: String, required: true },
-	skills: { type: String, required: true },
-	applicants: { type: Array, required: true }
-}, { timestamps: true });
+const JobSchema = new Schema(
+  {
+    postedBy: { type: DataTypes.ObjectId, required: true },
+    title: { type: String, required: true },
+    description: { type: String, required: true },
+    skills: [SkillSchema],
+    locations: [String],
+    vacancies: { type: Number, require: true },
+    isDeleted: { type: Boolean, default: false },
+  },
+  { timestamps: true }
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 2 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
	{ timestamps: true }
```